en/docs/chapter_sorting/quick_sort.md

```diff
@@ -91,9 +92,9 @@ Sample code is as follows:
 
 ## Tail recursion optimization
 
-**Under certain inputs, quick sort may occupy more space**. For a completely ordered input array, assume the sub-array length in recursion is $m$, each round of pivot partitioning produces a left sub-array of length $0$ and a right sub-array of length $m - 1$, meaning the problem size reduced per recursive call is very small (only one element), and the height of the recursion tree can reach $n - 1$, requiring $O(n)$ stack frame space.
+**Under certain inputs, quick sort may occupy more space**. For a completely ordered input array, assume the subarray length in recursion is $m$, each round of pivot partitioning produces a left subarray of length $0$ and a right subarray of length $m - 1$, meaning the reduction in problem size for each level of recursion is very small (only reducing by one element), and the height of the recursion tree can reach $n - 1$, requiring $O(n)$ stack frame space.
```

Comment:
occupy =>  take up 
Try to separate the long sentence into several short sentences to improve readability. Updated translaltions: 
Let's take a completely ordered input array as an example. Let the length of the subarray in the recursion be $m$. Each round of sentinel partitioning will produce a left subarray of length $0$ and a right subarray of length $m - 1$. This means that the problem size reduced by each layer of recursive calls is very small (only one element is reduced), and the height of the recursive tree can reach $n - 1$, which requires $O(n)$ of stack frame space.

Reply:
Thank you very much for your suggestions on improving the readability of the article. I made some slight modifications based on your advice.

Reply:
The last sentence has not been modified. Please have a check!

Reply:
Thanks for your reminding, it has been fixed